### ISSUE TITLE
User supplied cli.ini files don't seem to work

### ISSUE THREAD

**Issue by Crazee** [letsencrypt/letsencrypt#640]
As suggested by the documentation, I tried this, which failed:

$ ./venv/bin/letsencrypt --config examples/dev-cli.ini auth
usage: 
  letsencrypt [SUBCOMMAND] [options] [domains]

The Let's Encrypt agent can obtain and install HTTPS/TLS/SSL certificates.  By
default, it will attempt to use a webserver both for obtaining and installing
the cert.  
letsencrypt: error: unrecognized arguments: --config-dir /tmp/le/conf --work-dir /tmp/le/conf --logs-dir /tmp/le/logs --email foo@example.com --domains example.com --text --agree-eula --debug --verbose --authenticator standalone

**Comment by kuba** [comment]
This is a known upstream problem: https://github.com/bw2/ConfigArgParse/issues/26

**Comment by FloFra** [comment]
Downgrading to v0.9.3 of ConfigArgParse should resolve the issue till they fix the regression upstream.

If you followed the official installation documentation and installed the letsencrypt client under a virtual environment in ```venv/``` just run the following from inside the letsencrypt folder:
```
./venv/bin/pip install git+https://github.com/bw2/ConfigArgParse.git@0.9.3
```

**Comment by pde** [comment]
@FloFra feel free to send us a PR against `requirements.txt` if you can pin the old ConfigArgParse that way?

**Comment by kuba** [comment]
Old ConfigArgParse (0.9.3) doesn't support Python 2.6, that's why it's pinned this way (my branch with Py2.6 support). User-facing instructions now use PyPI only and skip `requirements.txt` so it's not that a big deal now, as long as you're not a developer.

**Comment by kuba** [comment]
More sensible solution is to pin https://github.com/kuba/ConfigArgParse/commit/a58b35d75a10e8b8fbee7f3c69163b63bb506325: my py2.6 compatible commit based on 0.9.3.

**Comment by kuba** [comment]
This will be fixed in #1257.
